src/Aspire.Cli/Projects/ProjectLocator.cs

```diff
@@ -38,39 +38,64 @@ internal sealed class ProjectLocator(ILogger<ProjectLocator> logger, IDotNetCliR
             };
 
             interactionService.DisplayMessage("magnifying_glass_tilted_left", InteractionServiceStrings.FindingAppHosts);
+            
+            // Search for both .csproj files and apphost.cs files
             var projectFiles = searchDirectory.GetFiles("*.csproj", enumerationOptions);
-            logger.LogDebug("Found {ProjectFileCount} project files in {SearchDirectory}", projectFiles.Length, searchDirectory.FullName);
+            var singleFiles = searchDirectory.GetFiles("apphost.cs", enumerationOptions);
```

Comment:
Do we need to worry about case sensitivity here? We don't generate the file with all lower case in our templates.